mws/resource_log_delivery.go

```diff
@@ -0,0 +1,131 @@
+package mws
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/databrickslabs/databricks-terraform/common"
+	"github.com/databrickslabs/databricks-terraform/internal"
+	"github.com/databrickslabs/databricks-terraform/internal/util"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+// LogDelivery wrapper
+type LogDelivery struct {
+	LogDeliveryConfiguration LogDeliveryConfiguration `json:"log_delivery_configuration"`
+}
+
+// LogDeliveryList lists log delivery configurations
+type LogDeliveryList struct {
+	LogDeliveryConfigurations []LogDeliveryConfiguration `json:"log_delivery_configurations"`
```

Comment:
whats LogDeliveryConfiguration(s) here? Not sure if it work.

Reply:
you're correct. i ended up not using list endpoint. will remove list type.